tutorials/tour/unified-types-diagram.dot

```diff
@@ -0,0 +1,13 @@
+digraph UnifiedTypes {
```

Comment:
This graph is misleading or incomplete in numerous ways, including
  1. Makes it look like AnyRef has only three children
  2. Makes it look like Null and Nothing are logically on the same level (they are not)
  3. Does not show the subtyping relationship between List and Seq
  4. Doesn't show conversions between primitives

As it stands the graph is worse than nothing.  It's really important to get this stuff right to start people off with an accurate mental model instead of a bunch of misconceptions that have to be unlearned.